js/buildAll.py

```diff
@@ -0,0 +1,96 @@
+import argparse
+import os
+import shutil
+import subprocess
+import sys
+from os.path import abspath, dirname
+
+build_script = "build.sh"
+if "win" in sys.platform:
+    build_script = "build.bat"
+
+build_dir = ["wasm", "wasm_SIMD", "wasm_threaded", "wasm_SIMD_threaded"]
+build_flags = ["", " --enable_wasm_simd", " --enable_wasm_threads", " --enable_wasm_simd --enable_wasm_threads "]
```

Comment:
please remove the space inside the flags. when you join those flags always add spaces